Description: remove SONAME
 libuid_wrapper is a preloadable library.
 We do not support directly linking to it, so it doesn't need the SONAME.
 Let's remove it.
Author: Jakub Wilk <jwilk@debian.org>, Jelmer Vernooij <jelmer@debian.org>
Last-Update: 2014-06-01

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,16 +16,6 @@ project(uid_wrapper VERSION 1.2.8 LANGUA
 # global needed variables
 set(APPLICATION_NAME ${PROJECT_NAME})
 
-# SOVERSION scheme: CURRENT.AGE.REVISION
-#   If there was an incompatible interface change:
-#     Increment CURRENT. Set AGE and REVISION to 0
-#   If there was a compatible interface change:
-#     Increment AGE. Set REVISION to 0
-#   If the source code was changed, but there were no interface changes:
-#     Increment REVISION.
-set(LIBRARY_VERSION "0.0.9")
-set(LIBRARY_SOVERSION "0")
-
 # add definitions
 include(DefinePlatformDefaults)
 include(DefineOptions.cmake)
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -15,10 +15,7 @@ target_link_libraries(uid_wrapper ${UIDW
 set_target_properties(
   uid_wrapper
     PROPERTIES
-      VERSION
-        ${LIBRARY_VERSION}
-      SOVERSION
-        ${LIBRARY_SOVERSION}
+      NO_SONAME ON
 )
 
 install(TARGETS uid_wrapper
